apps/server/server.cpp

```diff
@@ -620,6 +621,12 @@ void invoke_sdk_api(payload::ClientRequest buff_recv) {
         break;
     }
 
```

Comment:
Why do you need the server to tell what connection type it is? It can’t be anything else than Network.

Reply:
Ignore this. I misunderstood things.